lab_4_summarization_textrank/main.py

```diff
@@ -2,14 +2,21 @@
 Lab 4
 Summarize text using TextRank algorithm
 """
-from typing import Union
-
+from typing import Union, Iterable
+import re
 from lab_3_keywords_textrank.main import TextEncoder, \
     TextPreprocessor
 
 PreprocessedSentence = tuple[str, ...]
 EncodedSentence = tuple[int, ...]
 
+def check_iter(iterable: Iterable, val_type: type) -> bool:
+    """
+    Checks, if all values in iterable are of needed type
+    """
+    if not all(isinstance(val, val_type) for val in iterable):
```

Comment:
you can remove the result of `not all(isinstance(val, val_type) for val in iterable)` as is

Reply:
Done